data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,531 @@
+			`	It becomes clear you should have come with only the Ancient Ladybug, as the associates suggested.`
+				decline
+			label "not moded bug"
+			`You enter the system and the Unfettered shoot you? as usual. Perhaps they would have acted differently if you had used the Ancient Ladybug as a flagship, but it is too late to find out now.`
+				decline
+	on fail
+		"reputation: Hai (Unfettered)" <?= -1000
+		dialog `The Unfettered and their associates are clearly not happy when they somehow find out you did not get the cargo there in time. The message they send you is short: "You will have to prove yourself another way."`
+	on visit
+		dialog `You have reached <planet>, but you either did not bring the Ancient Ladybug, some of the cargo was on another ship, or you are not using the Ladybug as your flagship. You should park your fleet and only use the Ancient Ladybug.`
+	to complete
+		has "flagship model: Ancient Ladybug"
+	on complete
+		dialog `You receive a message from the 'associates of Hai-home': "Thank you for helping our brethren, <last>. We will not forget this. Please accept these <payment> to get you started on paying the gift you still need to prove yourself worthy of."`
+		payment 1000000
+		log "Delivered a pressing food cargo to the Unfettered using a newly acquired but rather old Ladybug. They seem to not be firing on me, for now at least."
+
+
+
+# no category, meaning you can't remove it from the Ladybug
+outfit "Unfettered Landing Permit"
+	"maintenance costs" 681
+	description "This item is needed in order to land on Unfettered planets, and its loss will cancel the arrangement."
+
+mission "Hai Legacy 3"
+	name "Ancient claim"
+	description "Beat the Lightning Bugs in a merciless fight. You only have one day, and becoming disabled will result in losing the fight."
+	landing
+	source "Firelode"
+	deadline 2
+	to offer
+		has "flagship model: Ancient Ladybug"
+		has "Hai Legacy 2: done"
+	on offer
+		conversation
+			`As you land and exit your ship, it is obviously the center of the spaceport crowd's attention, probably due to the ancient design. One Unfettered that looks like an engineer comes to speak to you, warmer than usual in your experiences with the Unfettered. "Welcome. Although I do not know how you have obtained this ship, I imagine some of us will see it as a sign. It is a relic of our past, a past we are quite proud of.`
+			`	"As such, most will say that you need to prove to us you have a right to use that ship. Luckily for you, I do not believe in any of that. What matters to me is that this ship is very durable, given the time it obviously spent in the water. It also has ancient technology that would be very valuable to us."`
+			`	Their tone suddenly changes as they raise their voice for everyone to hear, "I challenge you! The winner will earn the right to own this honorable relic!"`
+			choice
```

Comment:
The player doesn't necessarily know or care that the Unfettered work on an honor system where challenges like this are legit. For roleplay purposes, there should be an option for `<first>` to just say "no, I bought this and it's mine" and just leave. It can tie into the refusal later, but the roleplayer should also have that option from the beginning of the conversation.

Reply:
alright but at the very least they'll pursue you and worst case they'll shoot you right now x)

Reply:
The player is an outsider to the Unfettered, and you can't guarantee that the player will know the things you've just explained will happen. Perhaps if the player refuses, the leader mentioned later on steps in and explains to the player _why_ saying no and leaving isn't an option? Something like that would work while still allowing the conversation to progress as is currently written.

Reply:
well I added an option to get the heck out of there with Unfettered on your tails now and I kinda like it ~~you'll lose the ship shortly afterwards tho but that's smth else entirely~~